lib/app/item.ex

```diff
@@ -235,16 +235,36 @@ defmodule App.Item do
     %{name: nil, num_items: 3, num_timers: 8, person_id: 0}
     %{name: username, num_items: 1, num_timers: 3, person_id: 1}
   ]
+
+  iex> person_with_item_and_timer_count(:person_id, :desc)
+  [
+    %{name: username, num_items: 1, num_timers: 3, person_id: 1}
+    %{name: nil, num_items: 3, num_timers: 8, person_id: 0}
+  ]
   """
-  def person_with_item_and_timer_count() do
+  def person_with_item_and_timer_count(
+        sort_column \\ :person_id,
+        sort_order \\ :asc
```

Comment:
When running `mix c`, this is the single line that is not covered. This behaviour is weird, since it's definitely being covered in tests (see `test "Item.person_with_item_and_timer_count/2 returns a list sorted in ascending order when :asc is passed for sort_order" do`).

Have we had this happening before?

Reply:
@LuchoTurtle I found the issue!

We weren't testing the `person_with_item_and_timer_count/2` without passing the sort_order, so like:
`person_with_item_and_timer_count/1`

It's fixed now, I just committed :D